cds-snc/platform-forms-client
# feat: form response retrieval authentication api

# Summary | Résumé
As a form owner I would like to retrieve a temporary token so that I may authenticate against GC Form services such as the Form Response Retrieval API

closes #502 

# Test instructions | Instructions pour tester la modification


To test this API specifically:
- Open Postman UI
- The path for the API is: **/api/token/temporary**
- Payload sample (body / raw): **{ "email": "gc-forms@canada.ca" }**
- Authorization: Select type **Bearer Token**
- Enter the bearer token for the authorization
- Submit the request; the response should be: **{"message": "success"}**

Entering an incorrect payload, or incorrect bearer token should respond with error messages.

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

- Will need to send out a message via Notify upon temporary token generation. (#558)

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:
